### PR TITLE
Added NAT and service start bug fixes and new known issue to changelog

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added NAT and service start bug fixes and new known issue to changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,17 +10,27 @@
...
+
+New known issues: 
+
+- `Intrinsic gas exceeds gas limit` returned when calling `delete mapping[addr]` or `mapping[addr] = 0` [\#696](https://github.com/hyperledger/besu/issues/696)
+
+Calling delete and set to 0 Solidity mapping in Solidity fail.
+
 Previously identified known issues: 
 
 - [Eth/65 not backwards compatible](KNOWN_ISSUES.md#eth65-not-backwards-compatible)
```
Comment (shemnon) [inline CHANGELOG.md:31]: Should we link mitigations?  For this https://github.com/hyperledger/besu/pull/741 disabled ETH/65 by default.

**Comment by matkt** [comment]
Looks good for the NAT part

File: CHANGELOG.md (hunk 1 above)
Comment (timbeiko) [reply CHANGELOG.md:31]: Probably, yes, but perhaps in `KNOWN_ISSUES.md` ? I think we can add that PR to this line: https://github.com/hyperledger/besu/pull/754/files#diff-05319f6cacd1bf22bdbbf49f484df078L16

Comment (MadelineMurray) [reply CHANGELOG.md:31]: Added in known issues.  741 is also listed in the bug fixes.

**Pull request merged**